lib/hive.rb

```diff
@@ -63,6 +63,10 @@ def self.devicedb(section)
     @devicedb[section] ||= Object.const_get('DeviceDBComms').const_get(section).new()
   end
 
+  def self.hive_mind
```

Comment:
For consistency could you either change the method name to hivemind or the variable to @hive_mind. This may also require a change to bbc/hive-runner-android#40